### ISSUE TITLE
refactor mss_wms_wsgi.standalone_paste.py

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#30]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/30) by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

After we have copied the wsgi apache2 example to a python module wms.py in mslib.mswms we also can run it standalone by a very short script.

The wsgi script mainly needs only this call

```
#!python

from mslib.mswms.wms import application
```

The following example illustrates the call of the same code as on the apache2 server running but executed on localhost.

```
#!python

import sys

sys.path.extend(["$HOME.config/mss"])
from mslib.mswms.wms import application
import paste.httpserver
paste.httpserver.serve(application)
```

@wxmetvis I think about a deprecated warning for 1.1 in mss_wms_wsgi.standalone_paste.py and think about removing this in 1.2

As there may be more work on this, e.g. package dependencies I think we do that refactoring after the release of 1.1.

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

mss_wms_wsgi.standalone_paste replaced by mswms using wms, fixes #30

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Merged in ReimarBauer/mss (pull request #60)

mss_wms_wsgi.standalone_paste replaced by mswms using wms, fixes #30
